### ISSUE TITLE
Boxed tree display gets printed out in a broken format for Linux machine.

### ISSUE THREAD

**Issue by youngcc3157** [casey-c/egg#22]
Format of boxed tree display works fine in Windows Qt creator but format for Linux machine gets broken for some reason. It needs to be fixed.

**Comment by casey-c** [comment]
I can confirm this is a font issue. I set the font of the textEdit to Consolas (a fixed width mono-spaced font in Windows) but that font doesn't exist in Linux. I was able to fix the display issue in Linux by changing the font to something monospaced, such as Noto mono. Then the display is fine.

A couple of potential solutions:
1. Mess about the qmake to select different fonts per operating system
2. Figure out a way to tell the textEdit to pick a member of a monospaced font family regardless of what particular type is available on that particular operating system
3. Include the .ttf font file as a qresource (we'll have to find something with a creative commons / freely distributable license)

I'm thinking (3) is probably the easiest to implement, but will increase the size of our executable. (2) would definitely be the preferred route. And (1) just sounds scary.
